npm/debugTools/dataplaneConverter/converter.go

```diff
@@ -0,0 +1,297 @@
+package converter
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/Azure/azure-container-networking/npm/debugTools/dataplaneParser/iptable"
+	"github.com/Azure/azure-container-networking/npm/debugTools/dataplaneParser/parser"
+	"github.com/Azure/azure-container-networking/npm/debugTools/pb"
+	"github.com/Azure/azure-container-networking/npm/util"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+type Converter struct {
+	ListMap           map[string]string
+	SetMap            map[string]string
+	RequiredChainsMap map[string]bool
+}
+
+func (c *Converter) GetNpmCache(filename ...string) *NPMCache {
```

Comment:
This is a crucial function, if this fails we have nothing right, so lets return error and continue based on this error. You are doing that i am sure, but bubbling up error is always a good idea, so tomorrow if we want to have a mitigation in place in the caller function we can add something